AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -146,7 +146,7 @@ ecdc.europa.eu###cckBanner
 gidasanayim.com###cerezpolitikasi
 bechstein.com###cookieinformation
 1.fm##.gdprcookiescont
-zeit.de###gdpr_info_layer
+www-zeit-de.cdn.ampproject.org,zeit.de###gdpr_info_layer
```

Comment:
Please also add google.de to the rule. Else iOS will not hide it (rules for iframe urls not working on iOS).